dali/kernels/imgproc/convolution/convolution_gpu.h

```diff
@@ -29,6 +30,29 @@
 namespace dali {
 namespace kernels {
 
+struct ConvEpilogue {
+  ConvEpilogue(span<const float> scales, float beta = 0.f)  // NOLINT
+      : scales_{scales}, scale_{}, beta_{beta} {}
+  ConvEpilogue(float scale, float beta = 0.f) : scales_{}, scale_{scale}, beta_{beta} {}  // NOLINT
+
+  inline float num_samples() const {
+    return scales_.empty() ? 1 : scales_.size();
+  }
+
+  inline float alpha(int sample_idx) const {
+    return scales_.empty() ? scale_ : scales_[sample_idx];
+  }
+
+  inline float beta() const {
+    return beta_;
+  }
```

Comment:
I think all these functions should be marked as DALI_HOST_DEV

Reply:
Does it? Could you explain why?